form_designer/contrib/cms_plugins/form_designer_form/migrations/0001_initial.py

```diff
@@ -1,11 +1,20 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+import cms
 from django.db import migrations, models
+from pkg_resources import parse_version as V
 
+# Django CMS 3.3.1 is oldest release where the change affects.
+# Refs https://github.com/divio/django-cms/commit/871a164
+cmp = lambda x: (x[0] > x[1]) - (x[0] < x[1])
+if cmp((V(cms.__version__), V('3.3.1'))) >= 0:
```

Comment:
It doesn't have to be this complicated. The version objects returned by `V` can be compared directly:

```
if V(cms.__version__) >= V('3.3.1'):
    ...
```

Reply:
Hmm.. that is also true, good point! That is now changed.